### PR TITLE
Sequel benchmark to create string columns

### PR DESCRIPTION
**Pull request by shahsaurabh0605**

### CODE REVIEW

File: rails/benchmarks/bm_sequel_create_string_columns.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+COUNT=25 
+
+Sequel.migration do
+  change do
+    create_table(:users) do
+      primary_key :id
+      COUNT.times do |i|
+        column :"column#{i}", "varchar(255)"
+      end
```
Comment (tgxworld) [inline rails/benchmarks/bm_sequel_create_string_columns.rb]: I think we are missing the time stamp columns here.

File: rails/benchmarks/bm_sequel_create_string_columns.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    end
+  end
+end
+
```
Comment (tgxworld) [inline rails/benchmarks/bm_sequel_create_string_columns.rb:17]: Small nitpick, extra line here

**Comment by tgxworld** [comment]
What output do you get when you run the benchmark file?

**Comment by shahsaurabh0605** [comment]
Had a tough time solving the errors to get the output, so finally:
`{"label":"sequel/sqlite_create_string_columns","version":"5.1.0.alpha","iterations_per_second":1638.2596193728432,"iterations_per_second_standard_deviation":6.653402104956189,"total_allocated_objects_per_iteration":654}
`

**Comment by tgxworld** [comment]
@shahsaurabh0605 What errors were you getting? :thought_balloon: You need to make sure you can run each benchmark file.

**Comment by shahsaurabh0605** [comment]
To run the sequel benchmarks I have to run `RAILS_MASTER=1 DATABASE_URL=sqlite::memory: ruby benchmarks/bm_sequel_create_string_columns.rb`. So the adapter in `DATABASE_URL` is just `sqlite` instead of `sqlite3`. This lead to further complications but now its running.

**Comment by tgxworld** [comment]
Did you try with `postgres` and `mysql`? You can have a look at 

https://github.com/ruby-bench/ruby-bench-suite/blob/master/rails/benchmarks/driver.rb#L14-L17
to see how it is done.

**Comment by shahsaurabh0605** [comment]
I looked into that but could not figure out how to use it. What should I write in DATABASE_URL?

**Comment by shahsaurabh0605** [comment]
In the case of mysql I need to set the default `:null=>true` for time stamp columns. I get this output:
`{"label":"sequel/mysql2_create_string_columns","version":"5.1.0.alpha","iterations_per_second":21.765204440161668,"iterations_per_second_standard_deviation":18.377957399835427,"total_allocated_objects_per_iteration":320}
`

**Comment by tgxworld** [comment]
> In the case of mysql I need to set the default :null=>true for time stamp columns.

Why is this so :question: 

What about PostgreSQL?

**Comment by shahsaurabh0605** [comment]
For memory testing in mysql I had to create table `users` to run the benchmark file which initially has null in `created_at` and `updated_at` fields. So for testing I kept null=true. This problem won't happen in postgres. In postgres I get this output:
`{"label":"sequel/postgres_create_string_columns","version":"5.1.0.alpha","iterations_per_second":89.41151072618388,"iterations_per_second_standard_deviation":4.4736969183416475,"total_allocated_objects_per_iteration":353}`

**Comment by tgxworld** [comment]
Awesome! Merging in :)

**Pull request merged**